packages-common/pluggable-widgets-tools/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "./configs/tsconfig.base",
     "include": ["./src"],
+    "exclude": ["./src/**/__tests__"],
```

Comment:
why do we need this? I believe that appdev gets away without different test/non-test tsconfigs

Reply:
We don't need it indeed. Also we can just have one tsconfig, but since we are reusing the jest configs in the root of the mono repo, which are also used by widgets, we need to make two seperate files. Maybe we can get rid of these seperate files completely for widgets as well. But I would rather pick that up in another PR.

Reply:
It is indeed already mentioned in WT-2500. But that being sad, i don't see why is this change needed now. pluggable-widgets-tools doesn't reuse top-level jest or tsconfig and neither other way around.

Reply:
Changed it to one tsconfig